### PR TITLE
Create azure-cloud-cli.json

### PR DESCRIPTION
**Pull request by siraajul**
AZURE CLI v0.1

### CODE REVIEW

**Review by siraajul — COMMENTED** [review]
Title Changes

File: azure-cloud-cli.json [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+                    "definition": "কমান্ড লাইন দিয়ে লগিন ",
+                    "code": "az login -u myemail@address.com"
+                },
+                {
+                    "definition": "একাউন্ট লিস্ট করা ",
+                    "code": "az account list"
+                },
+                {
+                    "definition": "সাবস্ক্রিপশন সেট করা ",
+                    "code": "az account set --subscription "xxx" "
```
Comment (sabbirshawon) [inline azure-cloud-cli.json]: fix the issues

File: azure-cloud-cli.json [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+                    "definition":"লিনাক্স ভার্চুয়াল মেশিন বানানো ",
+                    "code":"az vm create --resource-group myResourceGroup --name myVM --image ubuntults"
+                },
+                {
+                    "definition":"উইন্ডোজ ভার্চুয়াল মেশিন বানানো",
+                    "code":"az vm create --resource-group myResourceGroup --name myVM --image win2016datacenter"
+                },
+                {
+                    "definition":"রিসোর্স গ্রুপ বানানো ",
+                    "code":"az group create --name myresourcegroup --location eastus
```
Comment (sabbirshawon) [inline azure-cloud-cli.json]: fix it

**Review by sabbirshawon — CHANGES_REQUESTED** [review]
fix the mentioned error and send pr again

**Review by siraajul — COMMENTED** [review]
Mentioned Error Fixed

**Review by siraajul — COMMENTED** [review]
error solved

**Pull request merged**